### DISCUSSION TITLE
Verifying something from metamask

### DISCUSSION THREAD

**Discussion by Noveleader** [smartcontractkit/full-blockchain-solidity-course-py#1000] [Q&A] [answered]
This question is very important to me:
Like if any company launches its NFT collection and gives a access to premium content(or anything else) only if you own their NFT.
So how to check in the user wallet that the particular NFT collection's NFT is present or not?
This is pretty much advanced to me, this must be possible right? How to do it?
Please help!!

**Comment by n4n0b1t3** [comment, marked as answer]
Curiosity is a good quality when you want to learn something new. But so is patience. Every journey begins with a small step. For example, [understanding what the first two lines in any Solidity contract mean](https://github.com/smartcontractkit/full-blockchain-solidity-course-py/discussions/998).
Jumping too quickly to more complex topics, like querying a blockchain to retrieve ownership of NFTs, can lead to frustration very quickly.
That's why I would suggest just putting a few more weeks into the basics, but don't forget your question, because you may already find the answer in this course.

**Reply by Noveleader** [reply]
Thanks, I know I am jumping to that so fast w/o enjoying the basics just wanted to know that it is possible or not, I think I got the answer.
I am focusing on learning solidity in a better way now, thanks pal lot of help in a day from your side.

**Reply by n4n0b1t3** [reply]
Anytime ;-)

**Reply by cromewar** [reply]
Nice explanation @n4n0b1t3 !!
